src/runtime/entitlements-manager.js

```diff
@@ -203,7 +212,9 @@ export class EntitlementsManager {
    * @return {!Entitlements}
    */
   parseEntitlements(json) {
-    const isReadyToPay = json['isReadyToPay'];
+    this.storage_.set(IS_READY_TO_PAY_STORAGE_KEY, json['isReadyToPay']);
+    const isReadyToPay =
```

Comment:
Why not just reverse these two lines and do `this.storage_.set(IS_READY_TO_PAY_STORAGE_KEY, isReadyToPay);`.

But also two issues here:
 1. I believe storage only accepts strings.
 2. That means that null/undefined will be stored as "null"/"undefined" which is not what you want. You can always remove from storage on null/undefined.